gems/aws-sdk-ec2/sig/aws-sdk-ec2/auth.rbs

```diff
@@ -10,8 +10,8 @@
 module AWS::SDK::EC2
   module Auth
     class Params < ::Struct[untyped]
-      attr_accessor operation_name (): ::Symbol
       attr_accessor region (): ::String
+      attr_accessor operation_name (): ::Symbol
```

Comment:
I think there's an ordering issue here. I tried using Tree Sort

Reply:
I'll fix this upstream in smithy-ruby.

Reply:
https://github.com/smithy-lang/smithy-ruby/pull/189